routemaster-drain.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency     'faraday', '>= 0.9.0'
   spec.add_runtime_dependency     'faraday_middleware'
   spec.add_runtime_dependency     'net-http-persistent', '< 3' # 3.x is currently incompatible with faraday
-  spec.add_runtime_dependency     'rack', '>= 1.6.2'
+  spec.add_runtime_dependency     'rack', '~> 1.4.5'
```

Comment:
We can't make this `~>` as Rails 5 needs Rack 2.x, so this will become incompatible with any new apps.

It needs to be `>=`.